projects/docs-tools/src/modules/module-info/module-info.module.ts

```diff
@@ -20,6 +23,7 @@ import { SkyDocsModuleInfoComponent } from './module-info.component';
     SkyCodeModule,
     SkyClipboardModule,
     SkyDefinitionListModule,
```

Comment:
Can we remove the `SkyDefinitionListModule` now?

Reply:
Good catch :) Fixed.